lib/cpp/src/thrift/TApplicationException.h

```diff
@@ -21,6 +21,7 @@
 #define _THRIFT_TAPPLICATIONEXCEPTION_H_ 1
 
 #include <thrift/Thrift.h>
+#include <stdint.h>
```

Comment:
We have `HAVE_STDINT_H` for this.
For older VC++, uint32_t is still available via windows/config.h, switching between boost and std, which is included via Thrift.h.
It's problematic for any non-windows without stdint.h, though.